### PR TITLE
Fix position bug and add the change event

### PR DESCRIPTION
**Pull request by FrancisKong**
When the relative element isn't the body element, then  the position of the selection box will go wrong.

### CODE REVIEW

**Comment by stephan281094** [comment]
Hi @FrancisKong, thank you for the PR. I'll try to find some time this week to review the code.

**Comment by FrancisKong** [comment]
Thank you for your time, I wrote a demo about the bug: [vue-drag-select-bug](https://github.com/FrancisKong/vue-drag-select-bug)

File: src/DragSelect.vue [hunk 1]
```diff
@@ -100,15 +106,21 @@
...
-          const elTop = boxB.top + window.scrollY
-          const elLeft = boxB.left + window.scrollX
+          const boxB = {
+            top: $el.offsetTop,
+            left: $el.offsetLeft,
+            width: $el.clientWidth,
+            height: $el.clientHeight,
+          };
+
+          const elTop = boxB.top
```
Comment (stephan281094) [inline src/DragSelect.vue]: The variables `elTop` and `elLeft` are no longer necessary. Use `boxB.top`/ `boxB.left` directly.

**Comment by FrancisKong** [comment]
I've udpated the code, you may merge now.

**Pull request merged**